client/src/utils/string-util.js

```diff
@@ -19,7 +19,7 @@ export function prettyPrintDateString(dateString) {
     if (!dateString) return;
 
     // Wednesday, October 16th, 2019 format
-    return moment(dateString).format('dddd, MMMM Do, YYYY');
+    return moment(dateString, "YYYY-MM-DD").format('dddd, MMMM Do, YYYY');
```

Comment:
I had to specify the format of the dateString otherwise Moment gave a deprecation warning. More info here: https://momentjs.com/guides/#/warnings/js-date/

Reply:
YYYY-MM-DD probably isn't on the list of formats it tries. Oh well.

Reply:
Yeah basically it's an ISO or RSA approved format so it falls back on using a JS Date which is soon to be deprecated.